app/controllers/admin/borroweds_controller.rb

```diff
@@ -0,0 +1,31 @@
+class Admin::BorrowedsController < AdminController
+  before_action :find_borrowed, only: %i(show update)
+
+  def index
+    @borroweds = Borrowed.page(params[:page]).per Settings.page_borrowed
+  end
+
+  def show
+    @borrowed_details = @borrowed
+                        .borrowed_details
+                        .includes(book: [:image_attachment, :image_blob])
+  end
+
+  def update
+    if @borrowed.update!(status: params[:status])
```

Comment:
```suggestion
    if @borrowed.update status: params[:status]
```
Sao lại dùng ! ở đây